docs/en/observability/view-observability-alerts.asciidoc

```diff
@@ -10,7 +10,7 @@ For more information on configuring alerts, see <<create-alerts-configure>>.
 ****
 
 The Alerts page lists all the alerts that have met a condition defined by a rule you created using the Logs,
-Metrics, Uptime, or APM apps.
+Infrastructure, Uptime, or APM apps.
```

Comment:
This is good but I do wonder if we shouldn't save ourselves future hassle and say "created using one of the Observability apps" in places like this?